src/hotspot/share/c1/c1_ValueMap.cpp

```diff
@@ -359,6 +359,28 @@ LoopInvariantCodeMotion::LoopInvariantCodeMotion(ShortLoopOptimizer *slo, Global
   }
 }
 
+class CheckInsertionPoint: public ValueVisitor {
```

Comment:
For consistency with other similar cases in `c1_ValueMap.cpp`: 
```suggestion
class CheckInsertionPoint : public ValueVisitor {
```